app/ui/src/components/user_messages.ts

```diff
@@ -36,16 +37,13 @@ export class UserMessages {
     }
 
     networkError(error: NetworkError | Error, message: String = 'An unexpected network error occurred. Please try again later') {
-        let description = ''
+        let description = message + '\n'
         if (error instanceof NetworkError) {
-            description = `(${error.statusCode})`
-                + ((error.errorDescription != null) ? " " + error.errorDescription : "")
-                + ((error.message != null) ? " " + error.message : "")
-        } else if (error instanceof Error) {
-            description = error.message
+            description += error.toString()
+        } else {
+            description += errrToString(error)
         }
-        this.error(message + (description ? `: ${description}` : '') + (error instanceof NetworkError &&
-        error?.stackTrace ? `\n${error.stackTrace}` : ''))
+        this.error(description)
```

Comment:
In `error` we can end the message with eclipses and have a more link to open the full error content on a new tab `<a href="data:text/html,<pre>full error</pre>">more</a>`

Reply:
7be27cea9539d5b1cc3e21df50b69b07b122a4d7